Terran-One/cw-simulate-ui
# Response Tab Overhaul

## Description
Overhaul response tab:
1. Prettier error messages
2. Data views/boxes scale to content size
3. Hide empty data views/boxes
4. Unify "empty tab" message in a common component

## Test steps
1. Create a new simulation
2. Instantiate contract
3. Run a couple executions (preferably with response data). Check out data views in response tab.
4. Run an erroneous execution. Check out error message in response tab.
